### DISCUSSION TITLE
Welcome to Turkish-Chicken-Wings Discussions!

### DISCUSSION THREAD

**Discussion by observableobject** [observableobject/Turkish-Chicken-Wings#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by observableobject** [comment]
@androidpirate @derekquinn 

We can use this space to collaborate on discussions that don't necessarily have an issue attached.  :brainstorm:

**Comment by androidpirate** [comment]
PR is up for traditional Turkish salads and recipes!

https://github.com/vfrascello/Turkish-Chicken-Wings/pull/4
